include/mbedtls/ctr_drbg.h

```diff
@@ -107,10 +107,10 @@
 /** The amount of entropy used per seed by default.
  *
  * This is 48 bytes because the entropy module uses SHA-512
- * #MBEDTLS_ENTROPY_FORCE_SHA256 is not set).
  *
  * \note See mbedtls_ctr_drbg_set_entropy_len() regarding what values are
  *       acceptable.
+ * (\c MBEDTLS_ENTROPY_FORCE_SHA256 is not set).
```

Comment:
In other cases you seem to prefer writing "is disabled" instead of "is not set".

Reply:
Since I was rebasing anyway, I amended the corresponding commit.